### ISSUE TITLE
Unable to delete message via webmail

### ISSUE THREAD

**Issue by bunyevacz** [nodemailer/wildduck-webmail#14]
When I click on delete (single message), the browser sends to the server:
```
https://myserver.com/api/delete (POST)
_csrf | dDdPbVpd-SaecvbSOOcjbKqZMNK0Ug4sHIBM
mailbox | 5b184657c4acb0049beafdc5
message | 4439
```

The response is:
`"Cannot read property 'value' of undefined"`

It fails at `/opt/wildduck-webmail/routes/api.js:router.post(/delete)` , 
at this code snippet:
```
} else {
            // move to trash
            apiClient.messages.update(
                req.user.id,
                result.value.mailbox,
                result.value.message,
                {
                    moveTo: trash.id
                },
                (err, response) => {
                    if (err) {
```

Now the `err` is true.

The complete error message:
```
 err: Error: Cannot read property 'value' of undefined
Sep 03 00:36:13 myserver.com wildduck-www[8771]:     at req.push (/opt/wildduck-webmail/lib/api-client.js:353:23)
Sep 03 00:36:13 myserver.com wildduck-www[8771]:     at parseResponse (/opt/wildduck-webmail/node_modules/restify-clients/lib/JsonClient.js:105:9)
Sep 03 00:36:13 myserver.com wildduck-www[8771]:     at IncomingMessage.done (/opt/wildduck-webmail/node_modules/restify-clients/lib/StringClient.js:207:13)
Sep 03 00:36:13 myserver.com wildduck-www[8771]:     at Object.onceWrapper (events.js:313:30)
Sep 03 00:36:13 myserver.com wildduck-www[8771]:     at emitNone (events.js:111:20)
Sep 03 00:36:13 myserver.com wildduck-www[8771]:     at IncomingMessage.emit (events.js:208:7)
Sep 03 00:36:13 myserver.com wildduck-www[8771]:     at endReadableNT (_stream_readable.js:1064:12)
Sep 03 00:36:13 myserver.com wildduck-www[8771]:     at _combinedTickCallback (internal/process/next_tick.js:138:11)
Sep 03 00:36:13 myserver.com wildduck-www[8771]:     at process._tickCallback (internal/process/next_tick.js:180:9) mailboxis: [ { id: '5b184657c4acb0049beafdc6',
```

I'm still hunting this bug down. It used to work (deleting message), but 
it could never delete one page of messages (20 messages), I think 16 message was the limit.
So I wanted to find that bug, instead now I can not even delete a single message.

**Comment by andris9** [comment]
Single message deletion was broken in latest version, I fixed it and pushed update as v1.4.6

**Comment by andris9** [comment]
That is wildduck@v1.4.6, not wildduck-webmail
